Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/util/StatisticsNumberValueFormatter.kt

```diff
@@ -0,0 +1,28 @@
+package de.rki.coronawarnapp.statistics.util
+
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Locale
+
+object StatisticsNumberValueFormatter {
```

Comment:
I'd prefer not having an object, either just define the func by itself or use a class and inject it with dagger

Reply:
I changed it to a top-level function.